src/osemgrep/language_server/Test_LS_e2e.ml

```diff
@@ -664,6 +669,9 @@ let check_startup info folders (files : Fpath.t list) =
   let%lwt () = assert_progress info "Refreshing Rules" in
   let%lwt rulesRefreshedNotif = receive_notification info in
   let%lwt () = assert_notif rulesRefreshedNotif "semgrep/rulesRefreshed" in
+  let rules_length = List.length info.server.session.cached_session.rules in
+  Alcotest.(check bool)
+    "rules refreshed greater than 1" (rules_length > 20) true;
```

Comment:
This is a test so it's probably fine but fyi we have a function to check if a list is shorter/longer than a fixed length if we don't need to count every element.